apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts

```diff
@@ -575,3 +575,55 @@ for (const transaction of transactionsAddressInvalid) {
     );
   });
 }
+
+test.describe("send NFT to ENS address", () => {
```

Comment:
this test fails because there is no account created, is it expected?

Reply:
PASS for me with latest commit